src/typespec.h

```diff
@@ -81,6 +81,13 @@ struct TypeSpec
 
     /// Get number of bytes required to store the field for a single point
     int size() const { return elsize*count; }
+
+    bool operator==(const TypeSpec& other) const
```

Comment:
Yep.  This is just the doxygen convention.  I'm not actually using doxygen, but I like the conceptual separation of three slashes for "public API" documentation from two slashes for implementation detail comments.